patches/tModLoader/Terraria/Social/Steam/WorkshopSocialModule.TML.cs

```diff
@@ -40,7 +40,12 @@ public override bool TryGetInfoForMod(TmodFile modFile, out FoundWorkshopEntryIn
 	public override bool PublishMod(TmodFile modFile, NameValueCollection buildData, WorkshopItemPublishSettings settings)
 	{
 		if (!SteamedWraps.SteamClient) {
-			base.IssueReporter.ReportInstantUploadProblem("tModLoader.SteamPublishingLimit");
+			IssueReporter.ReportInstantUploadProblem("tModLoader.SteamPublishingLimit");
+			return false;
+		}
+		
+		if (modFile.TModLoaderVersion != BuildInfo.tMLVersion) {
```

Comment:
please use `modFile.TModLoaderVersion.MajorMinor() != Buildinfo.tMLVersion.MajorMinor()`

Whether the build, patch version matches should not be a factor, as anything under the same MajorMinor should be compatible in Preview or Stable builds